### PR TITLE
Feature/navigation

### PR DESCRIPTION
**Pull request by ghost**
## Summary

* Install and configure react-navigation
* Create dummy screen to test the navigation

#### Pixel 3XL
image

#### iPhone 8
image

## Trello Card

[Trello Card](https://trello.com/c/Mh5dNlel/13-n-71-introducci%C3%B3n-a-react-navigation)

### CODE REVIEW

File: wbooks/src/app/index.tsx [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
   return (
-    <>
-      <StatusBar barStyle="dark-content" />
-      <LibraryScreen />
-    </>
+    <NavigationContainer>
+      <LibraryStack.Navigator initialRouteName={ROUTES.library}>
+        <LibraryStack.Screen name={ROUTES.library} component={LibraryScreen} />
+        <LibraryStack.Screen name={ROUTES.bookDetail} component={BookDetailScreen} />
+      </LibraryStack.Navigator>
```
Comment (sfernandez11) [inline wbooks/src/app/index.tsx]: Move this to `AppNavigator` component (create it):
```
<NavigationContainer>
  <StatusBar barStyle="dark-content" />
  <AppNavigator />
</NavigationContainer>
```
We use `AppNavigator` component for all the navigation logic and stacks/tabs navigators

**Review by sfernandez11 — CHANGES_REQUESTED** [review]
Great work, just one comment!

File: wbooks/src/app/index.tsx (hunk 1 above)
Comment (ghost) [reply wbooks/src/app/index.tsx]: Done!

File: wbooks/src/app/index.tsx [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
+import AppNavigator from '@components/AppNavigator';
 
 const App = () => {
   return (
-    <>
-      <StatusBar barStyle="dark-content" />
-      <LibraryScreen />
-    </>
+    <NavigationContainer>
+      <AppNavigator />
```
Comment (sfernandez11) [inline wbooks/src/app/index.tsx:10]: And the `StatusBar`?

Comment (ghost) [reply wbooks/src/app/index.tsx:10]: Hahaha done!

**Review by sfernandez11 — APPROVED** [review]
🥇

**Pull request merged**